js/interface.js

```diff
@@ -2610,11 +2693,48 @@ var DynamicLists = (function() {
           item.value = item.fieldValue;
         }
 
+
+        item.filterModifier = {
+          default: {
+            value: $('#date-field-' + item.id).val(),
+            offset: $('#number-field-' + item.id).val() ? $('#number-field-' + item.id).val() : null,
+            isDeviceTimeZoneEnabled: $('#enable-timezone-default-' + item.id).is(':checked')
+          }
+        };
+
         if (item.logic === 'empty' || item.logic === 'notempty') {
-          item.valueType = null;
+          item.valueType = '';
```

Comment:
I changed the value to an empty string due to the fact that all values of this type with a false value have an empty string, for the sake of purity in my opinion, this line was distinguished by the fact that it assigned a different value, although it carries the same functionality as all the others. I have tested and this change has no consequences. But I can change the given variable in reverse

In the case of null, it means that we do not know what a value is, it may or may not exist, we simply do not know. 
An empty string means that we know what the value is and that it is nothing.

Reply:
I changed this value for consistency. I tested everything works correctly